src/components/wall.js

```diff
@@ -1,4 +1,4 @@
-import {SIZES} from './beat';
+import { SIZES } from './beat';
```

Comment:
this one is weird. there should not be spaces around brackets

Reply:
It looks like that's how semistandard does it: https://github.com/standard/semistandard/blob/master/options.js